### ISSUE TITLE
Text contrast in Smashing Membership section

### ISSUE THREAD

**Issue by shadeed** [smashingmagazine/redesign#39]
**- Do you want to request a *feature* or report a *bug*?**

Bug.

**- What is the current behavior?**

In "Smashing Membership" block, the text "Already have an account? Log in!" is hard to read. I think the contrast could be better.

**- If the current behavior is a bug, please provide the steps to reproduce.**

1. Go to home page.
2. Scroll down to "Smashing Membership" section.
3. Try to read the text "Already have an account? Log in!".

**- What is the expected behavior?**

Text should be in a different color that is accessibile. According to [this](http://leaverou.github.io/contrast-ratio/#%238E2017-on-%23E33A2C) test.

**- Please mention your operating system version and the version of your browser**

All browsers. 

![screen shot 2017-03-17 at 8 39 21 am](https://cloud.githubusercontent.com/assets/8038659/24032241/9cf5e3ba-0aef-11e7-8b8b-91163dae0c17.jpg)

**Comment by vitalyfriedman** [comment]
The contrast has been approved — thanks for reporting it! @shadeed
